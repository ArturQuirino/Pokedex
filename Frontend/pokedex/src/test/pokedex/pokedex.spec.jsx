import React from 'react';
import Renderer from 'react-test-renderer';
import { act, cleanup } from '@testing-library/react';
import Pokedex from '../../pages/pokedex/pokedex';
import PokedexService from '../../pages/repositories/api';
import { mount } from 'enzyme';
import '../../setupEnzyme';

jest.mock('../../pages/repositories/api', () => {
    return jest.fn().mockImplementation(() => {
      return {obterTodosPokemons: jest.fn()};
    });
  });

beforeEach(() => {
});
afterEach(cleanup);

it('Deve renderizar corretamente', ()=> {
    const tree = Renderer.create(
        <Pokedex />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});