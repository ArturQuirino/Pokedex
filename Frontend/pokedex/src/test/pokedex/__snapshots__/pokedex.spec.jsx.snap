// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deve renderizar corretamente 1`] = `
<main
  className="pokedex-pagina"
/>
`;
